HISTORY.rst

```diff
@@ -54,6 +54,8 @@ Internal changes
 * Use numpy max and min in `properties_and_measures` (:pull:`76`).
 * Cast catalog date_start and date_end to "%4Y-%m-%d %H:00" when writing to disk. (:issue:`83`, :pull:`79`)
 * Skip test of coverage on the sum if the list of select files is empty. (:pull:`79`)
+* Added missing cf names in conversions.yml (:issue:`86`, :pull:`88`)
```

Comment:
```suggestion
* Added missing CMIP variable names in conversions.yml (:issue:`86`, :pull:`88`)
```
The variable codes are usually taken from CMIP6 in xclim. The CF conventions do not define those codes, only the standard names and "canonical" units.